javers-core/src/main/java/org/javers/core/metamodel/type/TypeFactory.java

```diff
@@ -42,23 +43,19 @@ JaversType create(ClientsClassDefinition def) {
     JaversType create(ClientsClassDefinition def, ClassScan scan) {
         if (def instanceof CustomDefinition) {
             return new CustomType(def.getBaseJavaClass());
-        } else
-        if (def instanceof EntityDefinition) {
+        } else if (def instanceof EntityDefinition) {
             return entityTypeFactory.createEntity((EntityDefinition) def, scan);
-        } else
```

Comment:
could you please redo the reformatting? it makes code review harder

Reply:
Do you have a formatting file that I can import in IntelliJ so I format code the same as you do?